lib/experimental/fonts/font-library/class-wp-rest-font-faces-controller.php

```diff
@@ -263,14 +256,12 @@ public function get_item( $request ) {
 		}
 
 		// Check that the font face has a valid parent font family.
-		$font_family = $this->get_font_family_post( $request['font_family_id'] );
+		$font_family = $this->get_parent_font_family_post( $request['font_family_id'] );
 		if ( is_wp_error( $font_family ) ) {
 			return $font_family;
 		}
 
-		$response = parent::get_item( $request );
-
-		if ( (int) $font_family->ID !== (int) $response->data['parent'] ) {
+		if ( (int) $font_family->ID !== (int) $post->post_parent ) {
 			return new WP_Error(
```

Comment:
We need to check the `$post->post_parent` directly here, since `$response->data['parent']` may not be in the response if the `_fields` param is used.